### PR TITLE
Cassandra metrics

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

File: shotover-proxy/src/transforms/cassandra/sink_single.rs [hunk 1]
```diff
@@ -106,13 +112,26 @@ impl CassandraSinkSingle {
 
                     let mut responses = Vec::with_capacity(expected_size);
                     let mut results = results?;
+                    tracing::info!("lskdfj");
```
[the remark this answers is not in the document]
Comment (conorbros) [reply shotover-proxy/src/transforms/cassandra/sink_single.rs:115]: doh

File: shotover-proxy/src/transforms/cassandra/sink_single.rs [hunk 2]
```diff
@@ -111,8 +117,20 @@ impl CassandraSinkSingle {
...
+                                                ..
+                                            }) = &message.original
+                                            {
+                                                counter!("cassandra_single_failed_request", 1, "chain" => self.chain_name.clone());
+                                            }
+                                        }
+                                        responses.append(&mut resp);
+                                    }
                                     (m, Err(err)) => {
+                                        counter!("cassandra_single_failed_request", 1, "chain" => self.chain_name.clone());
```
Comment (rukai) [inline shotover-proxy/src/transforms/cassandra/sink_single.rs:133]: where is this error coming from?

**Review by rukai — CHANGES_REQUESTED** [review]

File: shotover-proxy/src/transforms/cassandra/sink_single.rs (hunk 2 above)
Comment (conorbros) [reply shotover-proxy/src/transforms/cassandra/sink_single.rs:133]: Ah you're right, this is another internal shotover error. I'll remove this counter.

Comment (rukai) [reply shotover-proxy/src/transforms/cassandra/sink_single.rs]: did you reply to the wrong comment, this line is still here.

Comment (conorbros) [reply shotover-proxy/src/transforms/cassandra/sink_single.rs]: Fixed, thought I'd removed it.

**Review by rukai — APPROVED** [review]
lgtm!

**Review by benbromhead — APPROVED** [review]

**Pull request merged**